xcc/commands.py

```diff
@@ -94,6 +94,10 @@ def list_settings() -> Mapping[str, Any]:
 def set_setting(name: str, value: Union[str, int, bool]) -> str:
     """Sets the value of a setting.
 
+    To configure your Xanadu Cloud credentials, it is usually sufficient to run
```

Comment:
When wouldn't it be sufficient to do this? Could be nice to clarify this slightly.

Reply:
Good call! This should be sufficient for all non-Xanadu users so I've updated the text to avoid any confusion.